_posts/2022-10-05-pdf-generation-at-scale.md

```diff
@@ -0,0 +1,415 @@
+---
```

Comment:
`2022-10-05-pdf-generation-at-scale.md` => `2022-10-06-pdf-generation-at-scale-on-kubernetes.md`

Change date to today